src/js/utils/functions.js

```diff
@@ -0,0 +1,11 @@
+export const identity = (x) => x;
+
```

Comment:
* while this is written tersely, is it not a bit confusing to read and understand what it does?
    * would it be better to use `Array.find()` to return the first match with 'card.type === 'card'' `this.cards.map((card:any) => card.type.toLowerCase())`?
* is there a unit test for it?

Reply:
For an other time